bigquery/unit_tests/test_table.py

```diff
@@ -1129,9 +1133,12 @@ def test_fetch_data_w_alternate_client(self):
         table = self._makeOne(self.TABLE_NAME, dataset=dataset,
                               schema=[full_name, age, voter, score])
 
-        rows, total_rows, page_token = table.fetch_data(client=client2,
-                                                        max_results=MAX,
-                                                        page_token=TOKEN)
+        iterator = table.fetch_data(
+            client=client2, max_results=MAX, page_token=TOKEN)
```

Comment:
Sidenote...is this the preferred way to fit long signatures in the 79char limit? It seems like we mix how/when we do it like this. LMK and I'll do whichever way you say.